chore!: remove deprecated spec rule support (#2080)
Redocly/redocly-cli: docs/@v2/commands/lint.md

### Lint config file

The `lint` command also validates the configuration file.

> You may want to set severity level by using the `--lint-config` option.

Reviewer: One of the meanings of "may want to" is a suggestion so this is why I read it this way. :) Ok, so if you just meant that there is a possibility to set severity level, I would just say: You can set severity level by using the `--lint-config` option.

> You may want to set severity level by using the `--lint-config` option.

Reviewer: ```suggestion
You can set severity level by using the `--lint-config` option.
```
